src/main/java/com/stripe/StripeClient.java

```diff
@@ -570,4 +573,58 @@ StripeResponseGetterOptions buildOptions() {
           connectBase);
     }
   }
+
+  /**
+   * Send raw request to Stripe API. This is the lowest level method for interacting with the Stripe
+   * API. This method is useful for interacting with endpoints that are not covered yet in
+   * stripe-java.
+   *
+   * @param method the HTTP method
+   * @param relativeUrl the relative URL of the request, e.g. "/v1/charges"
+   * @param content the body of the request as a string
+   * @return the JSON response as a string
+   */
+  public StripeResponse rawRequest(
+      final ApiResource.RequestMethod method, final String relativeUrl, final String content)
+      throws StripeException {
+    RawRequestOptions options = RawRequestOptions.builder().build();
+    return rawRequest(method, relativeUrl, content, options);
+  }
+
+  /**
+   * Send raw request to Stripe API. This is the lowest level method for interacting with the Stripe
+   * API. This method is useful for interacting with endpoints that are not covered yet in
+   * stripe-java.
+   *
+   * @param method the HTTP method
+   * @param relativeUrl the relative URL of the request, e.g. "/v1/charges"
+   * @param content the body of the request as a string
+   * @param options the special modifiers of the request
+   * @return the JSON response as a string
+   */
+  public StripeResponse rawRequest(
+      final ApiResource.RequestMethod method,
+      final String relativeUrl,
+      final String content,
+      final RawRequestOptions options)
+      throws StripeException {
+    if (method != ApiResource.RequestMethod.POST && content != null && !content.equals("")) {
+      throw new IllegalArgumentException(
+          "content is not allowed for non-POST requests. Please pass null and add request parameters to the query string of the URL.");
+    }
+    return this.getResponseGetter()
+        .rawRequest(
+            new RawApiRequest(
+                BaseAddress.API, method, relativeUrl, content, options, options.getApiMode()));
+  }
+
+  /** Deserializes StripeResponse returned by rawRequest into a similar class. */
+  public StripeObject deserialize(String rawJson) throws StripeException {
+    if (rawJson == null) {
+      throw new IllegalArgumentException("rawJson cannot be null");
+    }
+
+    return StripeObject.deserializeStripeObject(
```

Comment:
pass a response getter in and use the overload that takes a string?

Reply:
Done. Added a new overload that doesn't require `Type`. Also added a test that the responsegetter flows through.